frontend/common/code-help/traits/traits-ios.js

```diff
@@ -0,0 +1,28 @@
+import Utils from '../../utils/utils';
```

Comment:
This is Javascript so I'm probably speaking out of turn here but I don't see where we're using the Utils class?